components/Note.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import Link from 'next/link'
+
+import { Note } from '../interfaces'
+
+type Props = {
+data: Note
+remove: any,
+seek: any,
+}
+
+function to_hms(t: number) {
+   let str = new Date(t * 1000).toISOString();
+   if(t < 60) {
+      return str.substr(17, 6);
+   } else if(t < 60*60) {
+      return str.substr(14, 9);
+   }
+   return str.substr(11, 12);
+}
+
+const NoteItem = ({ data, remove, seek }: Props) => (
+   <div className="noteitem">
+     <div className="text">{data.text}</div>
+     <Link href="#" ><a className="time" title="Go to time" onClick={(ev) => seek(ev, data.time)}>{to_hms(data.time)}</a></Link>
+     <Link href="#"><a className="remove" title="Remove Note" onClick={(ev) => remove(ev, data.id)}>x</a></Link>
+     <ul className="noteitemid">
+       <li>Note ID <b>{data.id}</b></li>
+       <li>VideoID <b>{data.video_id}</b></li>
+       <li>Time <b>{data.time.toFixed(3)}</b></li>
+     </ul>
+     <style jsx>{`
```

Comment:
just because we're using scss for this project, it would be ideal to move this into `Note.module.scss` and then `import` it at the top.

Reply:
Try to follow [this page](https://github.com/BotWCollective/botw-sr-website/wiki/Styling-Conventions) I've written for styling to ensure it's consistent.

Reply:
Moved style in `Note.module.scss`